lib/fetchtweets.py

```diff
@@ -47,18 +48,12 @@ def fetchtweets(logkeys,twitter_name,depth,tweetsfilename):
                 onefetch_count = depth-len(tweets_list)
     
         print(len(tweets_list),"tweets fetched in total")
+        
         #output to JSON
-        outputfile = open(tweetsfilename, "w") 
-        print("Output results to",tweetsfilename)
-        outputfile.write("[")
-        for status in tweets_list:
-            json.dump(status._json,outputfile,indent = 4)
-            if status == tweets_list[-1]:
-                break
-
-            outputfile.write(",\n")
-    
-        outputfile.write("]")
-        outputfile.close()
+        with open(tweetsfilename, "w") as outputfile:
+            # with statement automatically closes file handle as needed
+            print("Output results to {}".format(tweetsfilename))
+            json_list = [status._json for status in tweets_list]
```

Comment:
I know this is a better way. But I didn't figure out what's this"status._json", thus I didn't touch it.

Reply:
I think `json.dump()` will take care of it when it loops on the list, and you really don't need to concat it one by one. 

Also sorry for closing this PR due to a git-related issue.